### DISCUSSION TITLE
bot transfer to specific department

### DISCUSSION THREAD

**Discussion by GuilhermeDeLucca** [LiveHelperChat/livehelperchat#1894] [General]
I have a bot that screens visitors, so that they can be transferred!

and I have 5 different departments, how do I make the bot transfer to the specific department? OR even the department that visitor select?

I believe I am doing something very wrong! the other departments don't have a bot

https://doc.livehelperchat.com/docs/bot/transfer-bot-to-bot

https://forum.livehelperchat.com/viewtopic.php?id=4020

**Comment by remdex** [comment]
Hi,
You can either use https://doc.livehelperchat.com/docs/chat/start-chat-form-settings or just pass multiple departments for customer to choose the one.

From bot you can use just https://doc.livehelperchat.com/docs/bot/bot-update-current-chat#change-department
